benchmark/coverage_analyzer.py

```
@@ -0,0 +1,1134 @@
+import ast
+import json
+import os
+import sys
+import time
+import subprocess
+import threading
+import multiprocessing
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field, asdict
+from pathlib import Path
+from typing import Dict, List, Optional, Set, Tuple, Any, Union
```

Flake8 (7.2.0)

[error] 'typing.Union' imported but unused

(F401)